src/main/java/woowa/sw/baseball/BaseballGame.java

```diff
@@ -0,0 +1,85 @@
+package woowa.sw.baseball;
+
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * 숫자야구 게임 로직 class
+ * author : sw
+ */
+public class BaseballGame {
+    private User user;
+    private RandomBallGenerate randomBallGenerate;
+
+    public BaseballGame(User user, RandomBallGenerate randomBallGenerate) {
+        this.user = user;
+        this.randomBallGenerate = randomBallGenerate;
+    }
+
+    public void playBaseballGame() {
+        ResultCalcMachine resultCalcMachine = new ResultCalcMachine(generateRandomBalls());
+        StrikeBallInfo strikeBallInfo;
+        do {
+            strikeBallInfo = resultCalcMachine.resultCalc(generateUserBalls());
+            user.printResult(makePrintResultMsg(strikeBallInfo));
+        } while (resultCalcMachine.isThreeStrike(strikeBallInfo));
+
+        user.printEndMsg();
+        if (isContinueBaseballGame()) playBaseballGame();
```

Comment:
```suggestion
        if (isContinueBaseballGame()) {
            playBaseballGame();
        }
```
위와 같이 if문 내에 코드가 한 줄이어도 중괄호로 둘러싸는게 자바 컨벤션에 맞습니다.